systemtest/src/main/java/io/strimzi/systemtest/resources/imageBuild/ImageBuild.java

```diff
@@ -0,0 +1,192 @@
+     * @throws IOException
+     */
+    public static void buildImageOpenshift(String name, String namespace, String dockerfilePath, String imageTag, String baseImage) throws IOException {
+        String dockerfileContent = Files.readString(Paths.get(dockerfilePath), StandardCharsets.UTF_8);
+
+        BuildConfig buildConfig = new BuildConfigBuilder()
+            .withNewMetadata()
+                .withName(name)
+                .withNamespace(namespace)
+            .endMetadata()
+            .withNewSpec()
+                .withNewOutput()
+                    .withNewTo()
+                        .withName(name + ":" + imageTag)
+                        .withNamespace(namespace)
+                        .withKind("ImageStreamTag")
+                    .endTo()
+                .endOutput()
+                .withNewSource()
+                    .withType("Dockerfile")
+                    .withDockerfile(dockerfileContent)
+                .endSource()
+                .withNewStrategy()
+                    .withType("Docker")
+                    .withNewDockerStrategy()
+                        .addToBuildArgs(new EnvVar("BASE_IMAGE", baseImage, null))
+                .endDockerStrategy()
+                .endStrategy()
+            .endSpec()
+            .build();
+
+        ImageStream imageStream = new ImageStreamBuilder()
+                .withNewMetadata()
+                    .withName(name)
+                    .withNamespace(namespace)
+                .endMetadata().build();
+
+        BuildRequest buildRequest = new BuildRequestBuilder()
+            .withNewMetadata()
+                .withName(name)
+                .withNamespace(namespace)
+            .endMetadata()
+            .build();
+
+        ResourceManager.getInstance().createResourceWithoutWait(imageStream);
+        ResourceManager.getInstance().createResourceWithoutWait(buildConfig);
+        BuildConfigResource.buildConfigClient().inNamespace(namespace).withName(name).instantiate(buildRequest);
+
+        BuildUtils.waitForBuildComplete(name, namespace);
+    }
+
+    /**
+     * Create config map with Dockerfile loaded by Kaniko
+     * @param namespace location of the config map
+     * @param configMapName name of the config map
+     * @param dockerfilePath path to the Dockerfile
+     * @throws IOException
+     */
+    private static void createDockerfileConfigMap(String namespace, String configMapName, String dockerfilePath) throws IOException {
+        String dockerfileContent = Files.readString(Paths.get(dockerfilePath), StandardCharsets.UTF_8);
+
+        ConfigMap configMap = new ConfigMapBuilder()
+                .withNewMetadata()
+                .withName(configMapName)
+                .withNamespace(namespace)
+                .endMetadata()
+                .addToData("Dockerfile", dockerfileContent)
+                .build();
+
+        ResourceManager.getInstance().createResourceWithoutWait(configMap);
```

Comment:
Why it is without wait here as well?

Reply:
It as just due to testing process. Updated to use wait method.